docs/model/index.md

```diff
@@ -0,0 +1,36 @@
+# farmOS Data Model
+
+## Goals
+
+The goal of the farmOS data model is to represent and organize a wide variety
+of agricultural and ecological systems to allow for easy access and analysis.
+This ranges from small-scale garden records to industrial agriculture to
+national forestry management.
+
+farmOS can model data that was recorded via manual data entry, as well as data
+streams from sensors or other applications.
+
```

Comment:
This discusses data ingress via manual and automated means. Should it also discuss data egress? e.g. "via a user friendly UX and documented canonical APIs"

Reply:
My intention here was to highlight that the farmOS data model itself covers both types of data (eg: logs and data streams). I wasn't really thinking in terms of "input" vs "output", although maybe it should be reworded to make this more clear? The model itself doesn't need to concern itself with input/output methods, IMO - that can be described in the user guide and API docs.

Reply:
Interesting... makes sense. Though I'd argue there's a bit of asymmetry here where you're saying a specific way the data model is designed to support recording data, but not saying anything a specific about how the data model is designed to support "easy access and analysis". Not a big deal either way, just pointing it out.

Reply:
Yea I see what you mean... maybe the word "record" is the problem, because it connotes "ingress". Maybe rephrasing to use the word "represent" is a better choice. Something like:

> farmOS can represent both high-level metadata records about assets and events on a farm, as well as raw time-series data from sensors or other applications.

Still needs some phrasing work... but the key point I'm trying to make is that there are two types of data.

*Typically* the former is user-entered and the latter is automatically collected, but that's not always the case. So this particular sentence isn't so much about ingress/egress as it is about the types of data. Probably still needs some more wordsmithing to make that clear. :-)

Reply:
Another motivation for including this sentence is for newcomers who find this doc and are wondering things like "what can I record with farmOS?" / "does it *only* represent one or the other?" - I feel like this kind of question has come up a lot.